package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "testswitch-frontend-candidate",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "dev": "next",
+    "build": "next build",
+    "start": "next start",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/techswitch-learners/testswitch-frontend-candidate.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/techswitch-learners/testswitch-frontend-candidate/issues"
+  },
+  "homepage": "https://github.com/techswitch-learners/testswitch-frontend-candidate#readme",
+  "dependencies": {
+    "@zeit/next-css": "^1.0.1",
```

Comment:
I _think_ we might not need this one...

Reply:
ahh yeah sorry that one is my bad, i accidently opened the css guide and blindly did that

Reply:
have removed